.gitignore

```diff
@@ -26,6 +27,9 @@ target/
 /.settings/
 /.project
 /.classpath
+/codepipelines-ci/.classpath
+/codepipelines-ci/.project
+/codepipelines-ci/.settings/
 /janusgraph-*/.classpath
```

Comment:
These exclusions aren't necessary since they're covered under the below janusgraph-* patterns